test/core/security/security_connector_test.cc

```diff
@@ -373,8 +373,8 @@ class TestDefafaultSllRootStore : public DefaultSslRootStore {
 }  // namespace
 }  // namespace grpc_core
 
-// TODO: Convert this test to C++ test when security_connector implementation
-// is converted to C++.
+// TODO(unknown): Convert this test to C++ test when security_connector
```

Comment:
This TODO can be assigned to me and Juanli.

Reply:
Done